### ISSUE TITLE
Provide Darwin/arm64 build

### ISSUE THREAD

**Issue by hazcod** [hadolint/hadolint#625]
- [] This is a bug report
- [x] This is a feature request
- [ ] I searched existing issues before opening this one

### Expected behavior
hadolint should run natively on the latest macOS hardware. (M1 processor, arm64 architecture)

### Actual behavior
hadolint fails to install via `brew` since it does not have a darwin/arm64 build available.

### Steps to reproduce the behavior
```shell
% brew install hadolint
Error: hadolint: no bottle available!
```

**Comment by lorenzo** [comment]
duplicate of #411
